pkg/pubsub/pubsub.go

```diff
@@ -23,6 +23,7 @@ func NewCrawlerPubSub(projectID string, topic string) (*CrawlerPubSub, error) {
 // CrawlerPubSubMessage is the message sent from the crawler
 type CrawlerPubSubMessage struct {
 	NewsroomException bool   `json:"newsroomException"`
+	MultiSigException bool   `json:"multiSigException"`
```

Comment:
Didn't see it yet, but is there code in the processor to handle this?

Reply:
going to add to processor as soon as this goes in